assets/js/components/Dashboard/AlertsPage.tsx

```diff
@@ -260,6 +262,11 @@ const AlertsList: ComponentType<AlertsListProps> = ({
             ).length;
           }
 
+          let showAnimationOnMount = false;
+          if (prevAlertIds?.length) {
+            showAnimationOnMount = !prevAlertIds.includes(alert.id);
+          }
```

Comment:
If you make the above change to have `prevAlertIds` always be an array, this can be:
```suggestion
          const showAnimationOnMount = !prevAlertIds.includes(alert.id);
```